docs/protocol/src/addresses_keys/addresses.md

```diff
@@ -8,6 +8,14 @@ attaches a *detection key* to each address, allowing a user to outsource
 probabilistic transaction detection to a relatively untrusted third-party
 scanning service.
 
+## Privacy Implications
```

Comment:
Added privacy implications associated with detection keys based on comments made in component **Y**.